KalturaWowzaServer/src/main/java/com/kaltura/media/server/wowza/CuePointsManager.java

```diff
@@ -283,37 +277,53 @@ public void init() throws KalturaManagerException {
 	}
 
 	@Override
-	public void onEvent(IKalturaEvent event){
+	public void onEvent(IKalturaEvent event) {
 
-		if(event.getType() instanceof KalturaMediaEventType){
+		if (event.getType() instanceof KalturaMediaEventType) {
 
-			switch((KalturaMediaEventType) event.getType())
-			{
+			switch ((KalturaMediaEventType) event.getType()) {
 				case APPLICATION_INSTANCE_STARTED:
```

Comment:
Why are some events registered in KalturaCuePointsManager and some in CuePointsManager?